src/js/renderers/NavigatorLayout.js

```diff
@@ -34,33 +41,6 @@ export const GutenbergNavigatorlLayoutRenderer = ({
   renderers,
   cells
 }) => {
-    /*
-     * Recursive method to get all the `object` type property and return them as { path, key }
-     * Note: so far the method skips the root node, and only looking inside the `properties` prop
-    */
-   // nit: move this method into seperate file
-    const getObjectProperties = (obj, parentPath = '') => {
```

Comment:
There is no longer need for this util, since now the nested prop will generated the inner content and pass it to root via Context